ingestion/src/metadata/ingestion/source/database/snowflake.py

```diff
@@ -61,6 +61,46 @@ class SnowflakeSource(CommonDbSourceService):
     def __init__(self, config, metadata_config):
         super().__init__(config, metadata_config)
 
+    def get_all_table_tags(self):
+        results = self.connection.execute(
+            FETCH_SNOWFLAKE_ALL_TAGS.format(
+                self.config.serviceConnection.__root__.config.database
+            )
+        )
+        self.all_table_tags = {}
+        self.all_column_tags = {}
+        for result in results:
+            row = list(result)
+            tag_dict = {
+                "tag_id": row[2],
+                "tag_name": row[3],
+                "tag_value": row[4],
+                "tag_domain": row[10],
+            }
+            tag = OMetaTagAndCategory(
+                category_name=CreateTagCategoryRequest(
+                    name=tag_dict["tag_name"],
+                    description="SNOWFLAKE COLUMN TAG NAME",
+                    categoryType="Descriptive",
+                ),
+                category_details=CreateTagRequest(
+                    name=tag_dict["tag_value"], description="SNOWFLAKE COLUMN TAG VALUE"
+                ),
+            )
+            yield tag
+            if tag_dict["tag_domain"] == "TABLE":
+                table_name = row[8]
+                if self.all_table_tags.get(table_name):
+                    self.all_table_tags[table_name].append(tag_dict)
+                else:
+                    self.all_table_tags[table_name] = [tag_dict]
+            else:
+                column_name = row[12]
+                if self.all_column_tags.get(column_name):
+                    self.all_column_tags[column_name].append(tag_dict)
```

Comment:
What happens when there are two tables which has sample column names 🤔 
For ex. `Employee (id, name, age)` and `Company(id, name, regno)`

Tags applied on `Employee.id` & `Company.id` will be clubbed and applied on both 🤔 ?